### PR TITLE
feat(ui, llc): slow mode

### PR DESCRIPTION
**Pull request by deven98**
Adds slow mode

### CODE REVIEW

File: packages/stream_chat/lib/src/client/channel.dart [hunk 1]
```diff
@@ -201,6 +201,21 @@ class Channel {
...
+    return state?._channelState.channel?.cooldown;
+  }
+
+  /// Cooldown count as a stream
+  Stream<int?>? get cooldownStream {
+    _checkInitialized();
+    return state?.channelStateStream.map((cs) => cs.channel?.cooldown);
+  }
+
+  ///
```
Comment (deven98) [inline packages/stream_chat/lib/src/client/channel.dart:216]:
```suggestion
  /// Stores time at which cooldown was started
```

**Comment by imtoori** [comment]
@deven98 I think it's possible to add a couple of tests, especially in the llc

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart [hunk 2]
```diff
@@ -344,4 +344,7 @@ class StreamChatLocalizationsKo extends GlobalStreamChatLocalizations {
 
   @override
   String get replyToMessageLabel => '메시지에 회신합니다.';
+
+  @override
+  String get slowModeOnLabel => '느린 모드 켜짐';
```
Comment (sachaarbonel) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart:349]:
```suggestion
  String get slowModeOnLabel => '슬로모드온 입니다.';
```
@EricSSartorius thoughts?

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_ja.dart [hunk 3]
```diff
@@ -346,4 +346,7 @@ class StreamChatLocalizationsJa extends GlobalStreamChatLocalizations {
 
   @override
   String get replyToMessageLabel => 'メッセージに返信';
+
+  @override
+  String get slowModeOnLabel => 'スローモードオン';
```
Comment (sachaarbonel) [inline packages/stream_chat_localizations/lib/src/stream_chat_localizations_ja.dart:351]: LGTM. @EricSSartorius ?

**Review by sachaarbonel — CHANGES_REQUESTED** [review]

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_ja.dart (hunk 3 above)
Comment (EricSSartorius) [reply packages/stream_chat_localizations/lib/src/stream_chat_localizations_ja.dart:351]: yeah that works

File: packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart (hunk 2 above)
Comment (EricSSartorius) [reply packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart:349]: 슬로모드 켜짐 Might be best here

Comment (imtoori) [reply packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart:349]:
```suggestion
  String get slowModeOnLabel => '슬로모드 켜짐';
```

**Review by imtoori — APPROVED** [review]
LGTM

**Pull request merged**